cli/src/commands/extensions/extension.rs

```diff
@@ -61,9 +61,14 @@ impl Extension {
             let dest_path = target_prefix.join(source_path.strip_prefix(&self.path)?);
 
             if source_path.is_dir() {
-                fs::create_dir_all(&dest_path)?;
-                #[cfg(unix)]
-                fs::set_permissions(&dest_path, Permissions::from_mode(0o700))?;
+                if cfg!(unix) {
```

Comment:
Is it better to use DirBuilder in both cases? I guess it has the same effect.

```rust
let mut builder = DirBuilder::new()
    .recursive(true);

#[cfg(unix)]
builder.mode(0o700);

builder.create(&dest_path)?;
```

Reply:
I thought about that too -- I didn't particularly like stylistically having to split code this way nor replicating `DirBuilder` without mode on one branch, so I went for the current solution, but I do think yours has a more consistent semantics at least.